In [ ]:
# mount data
from google.colab import drive
drive.mount('/content/drive')

In [ ]:
df = pd.read_csv('https://archive.ics.uci.edu/ml/machine-learning-databases/abalone/abalone.data')

df.head()

In [2]:
from google.colab import drive
drive.mount('/content/drive')
# import libraries
import matplotlib.pyplot as plt
import seaborn as sns
import pandas as pd
import numpy as np

df = pd.read_csv('/content/latest_RAPTOR_by_player.csv')

df.head()

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


player_name  player_id  ...  predator_total  pace_impact
0   Precious Achiuwa  achiupr01  ...       -3.342885     0.329157
1       Jaylen Adams  adamsja01  ...      -16.978299    -1.834433
2       Steven Adams  adamsst01  ...        0.604871    -1.226775
3        Bam Adebayo  adebaba01  ...        1.590909     0.006968
4  LaMarcus Aldridge  aldrila01  ...       -2.654506    -0.432648

[5 rows x 21 columns]

In [3]:
len(df)

540

For this task:

Create a Colab notebook where you have uploaded and shown the .head() of each of your data sets.  For each of the proposed datasets, answer the following questions:

First choice: dataset 1

1) Source of data

538

2) Brief description of data

Data on which NBA players provide the most value based on a variety of different metrics(ie gauging which are the best players) 

3) What is the target?

raptor_total

4) Is this a classification or regression problem?

much more regression

5) How many features?

5-10 depending on the particular question addrssed at each point during the data 

6) How many rows of data.

540

7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?

will have to do a little more research on my own to understand the basis behind each of these metrics to better make sense of the mand figure out how to best design analysis between and amongst them. This data doesnt really have any classification component to it which is fine but that might lead to easier to understand answers/questions so extra detail needs to be paid to really being able ot describe and explain all the different components of this data set and what is going on. 

Also this isnt a big deal but they only have one year sample sizes per data set(as far as I am aware right now). If I want to do multi year studies I have to import that data separately which for presentability purposes isnt as optimal

In [4]:
df2 = pd.read_csv('/content/graduates.csv')

df2.head()

Year  ...  Employment.Work Activity.Teaching
0  1993  ...                             381908
1  1993  ...                              17718
2  1993  ...                             104191
3  1993  ...                              26892
4  1993  ...                             234507

[5 rows x 50 columns]

In [5]:
len(df2)

517

For this task:

Create a Colab notebook where you have uploaded and shown the .head() of each of your data sets.  For each of the proposed datasets, answer the following questions:

2nd choice: dataset 2

1) Source of data

corgis-edu

2) Brief description of data

for certain majors, what was their median salary, how many ended up working in the field, how many got graduate degrees, if they left the field what was the reason why, demographics of each major etc 

3) What is the target?

depends on what you want to study but mean salary is one idea 

4) Is this a classification or regression problem?

regression

5) How many features?

10-20

6) How many rows of data.

540

7) What, if any, challenges do your foresee in cleaning, exploring, or modeling with this dataset?

A ton of rows of data of which I might not have much use for so alot to sort through. No categorical data which in some cases with some of these variables might have been a little helpful. Alot of these variables like 
"Employment.Work Activity.Productions/Operations/Maintenance" Im going to need to take time to understand what htey really mean and itll take effort to be able to explain clearly what they are in a discussion of the project.